keel-clouddriver/src/test/kotlin/com/netflix/spinnaker/keel/clouddriver/MemoryCloudDriverCacheTest.kt

```diff
@@ -85,8 +85,31 @@ object MemoryCloudDriverCacheTest {
     }
   }
 
+  @Test
+  fun `subsequent requests for the same security groups are served from the cache`() {
+    every {
+      cloudDriver.getCredential("prod")
+    } returns Credential("prod", "aws")
+    every {
+      cloudDriver.getSecurityGroupSummaryByName("prod", "aws", "us-east-1", "foo")
+    } returns sg1
+
+    subject.securityGroupByName("prod", "us-east-1", "foo")
+    subject.securityGroupByName("prod", "us-east-1", "foo")
+
+    verify(exactly = 1) {
+      cloudDriver.getSecurityGroupSummaryByName(any(), any(), any(), any())
+    }
+    verify(exactly = 1) {
+      cloudDriver.getCredential(any())
+    }
+  }
```

Comment:
Incredibly it looks like there weren't any tests to ensure the cache was actually used